guides/common/modules/proc_preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage.adoc

```diff
@@ -0,0 +1,34 @@
+[id="preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage"]
+= Preparing for disaster recovery with active and passive {ProjectServer} with external storage
+
+Create your passive {ProjectServer} as a clone of your active {ProjectServer}.
+Make sure that both servers connect to the database content on your shared storage.
```

Comment:
What does "database content" mean here?

Reply:
I was trying to come up with a short way to describe `/var/lib/pulp` and `/var/lib/pgsql`. What's a good alternative here? Just "content"? Or just "data"? Or perhaps be specific like this:

```suggestion
Ensure that both servers connect to the contents of `/var/lib/pulp` and `/var/lib/pgsql` on your shared storage.
```

Reply:
How about
```suggestion
Ensure that `/var/lib/pulp` and `/var/lib/pgsql` on your shared storage is available to both servers.
```